### DISCUSSION TITLE
OAP-server support trace-ignore

### DISCUSSION THREAD

**Discussion by teamoTNT** [apache/skywalking#10721] [Ideas]
In the Java agent, we can use trace-ignore-plugin to ignore some specified endpoints, but not on the OAP-server. So can the OAP-server support this feature? 
I hope it can ignore the endpoints in the global scope or like sampling rate. 
When endpoints are ignored, OAP-server is no need to store and compute any metrics.

Background, there are always people or new services or other things that cause a lot of useless trace to increase storage and waste  resources.

**Comment by wu-sheng** [comment]
No, you can't. That costs too much to do endpoint matching.

**Reply by teamoTNT** [reply]
all right
